values/prometheus-operator/rules/gitea-db-backup.yaml

```diff
@@ -2,7 +2,7 @@ groups:
   - name: gitea-db-backup
     rules:
       - alert: BackupGiteaDbFailing
-        expr: time() - cnpg_collector_last_failed_backup_timestamp{namespace="gitea",pod=~"gitea-db-1"} > 86400 or time() - cnpg_collector_last_failed_backup_timestamp{namespace="gitea",pod=~"gitea-db-2"} > 86400
```

Comment:
Unfortunately offset does not work if there are no metrics from last 24H, it seems this is a default setting for the platform. Moreover DB cluster can have more than two replicas and that expression would not cover that use case.
@Ani1357 and I are proposing the following expression:
```
(time() - max(cnpg_collector_last_available_backup_timestamp{namespace="gitea"}) > 24 * 60 * 60)
```

Reply:
updated to the proposed solution